package.json

```diff
@@ -24,12 +24,9 @@
     "lint:fix": "nr lint --fix",
     "publish-ci": "tsx scripts/publish-ci.ts",
     "release": "bumpp package.json packages/*/package.json --commit --push --tag && git update-ref refs/heads/release refs/heads/main && git push origin release",
-    "test": "vitest --api -r test/core",
-    "test:run": "vitest run -r test/core",
-    "test:all": "CI=true pnpm -r --stream run test --allowOnly",
-    "test:ci": "CI=true pnpm -r --reporter-hide-prefix --stream --filter !test-browser --filter !test-esm --filter !test-browser run test --allowOnly",
-    "test:ci:vm-threads": "CI=true pnpm -r --reporter-hide-prefix --stream --filter !test-coverage --filter !test-single-thread --filter !test-browser --filter !test-esm --filter !test-network-imports --filter !test-browser --filter !example-react-testing-lib-msw run test --allowOnly --pool vmThreads",
-    "test:ci:no-threads": "CI=true pnpm -r --reporter-hide-prefix --stream --filter !test-vm-threads --filter !test-coverage --filter !test-watch --filter !test-bail --filter !test-esm --filter !test-browser run test --allowOnly --pool forks",
```

Comment:
This reduces CI runtimes a lot. So in future we'll need to be sure to add tests that loop through all pools that could affect the feature/fix that's being tested. I think at least `--bail` and coverage tests are already doing this.

Reply:
Yes. If the feature can be tested without a config change, it can also just be put in a test/core folder

Reply:
Also, this PR doesn't really change how we write tests. We had a lot of filters to remove certain folders - instead, I put all tests into test/core and added a workspace config to run them with all pool options. All tests outside test/core run Vitest with `runVitest` utility that requires the pool anyway, so this had no affect on them.